descope/management/fga.py

```diff
@@ -0,0 +1,140 @@
+from datetime import datetime, timezone
+from typing import Any, List, Optional
+
+from descope._auth_base import AuthBase
+from descope.management.common import MgmtV1
+
+
+class FGA(AuthBase):
+    def save_schema(self, schema: str):
```

Comment:
Are we missing here the "upgrade" flag (we have in lower level) ?

Reply:
We always upgrade for FGA